src/Illuminate/Http/Response.php

```diff
@@ -45,7 +45,7 @@ public function __construct($content = '', $status = 200, array $headers = [])
      *
      * @throws \InvalidArgumentException
      */
-    public function setContent($content)
+    public function setContent(mixed $content): static
```

Comment:
need to cast to a string or null before using the value

Reply:
> need to cast to a string or null before using the value

But that's not true, right? It can be anything: any value in `shouldBeJson`, `Renderable`, ...